### PR TITLE
add docstring for turbomind

### PR DESCRIPTION
**Pull request by lvhan028**

### CODE REVIEW

File: lmdeploy/model.py [hunk 1]
```diff
@@ -6,25 +6,38 @@
...
 
     @property
     def stop_words(self):
+        """Return the stop-word's token ids."""
         return None
 
 
 @MODELS.register_module(name='internlm')
 class InternLM:
+    """Chat template for InternLM model."""
```
Comment (grimoire) [inline lmdeploy/model.py:40]: 'for' here and 'of' in Llama model

File: lmdeploy/serve/client.py [hunk 2]
```diff
@@ -7,12 +7,22 @@
 
 
 def input_prompt():
+    """Input a prompt in the consolo interface."""
```
Comment (grimoire) [inline lmdeploy/serve/client.py:10]: console?

File: lmdeploy/model.py [hunk 3]
```diff
@@ -44,20 +67,33 @@ def get_prompt(self, prompt, sequence_start=True):
 
     @property
     def stop_words(self):
+        """Return the stop-word's token ids."""
```
Comment (AllentDan) [inline lmdeploy/model.py:70]: stop-word's -> stop-words'

File: lmdeploy/serve/turbomind/chatbot.py [hunk 4]
```diff
@@ -258,17 +260,21 @@ def cancel(self, session_id: int, *args, **kwargs):
...
         token_ids, _ = self.preprocess('<BOS>')
         return token_ids[0][0]
 
     def _get_eos(self):
+        """return eos token id."""
         token_ids, _ = self.preprocess('<EOS>')
         return token_ids[0][0]
 
     def _stop_words(self, stop_words: List[int]):
+        """return stop-word's token id."""
```
Comment (AllentDan) [inline lmdeploy/serve/turbomind/chatbot.py:277]: stop-word's -> stop-words'
id -> ids

File: lmdeploy/model.py [hunk 5]
```diff
@@ -6,25 +6,38 @@
...
+            str: the concatenated prompt
+        """
         if sequence_start:
             return f'{self.system} {self.user}: {prompt} {self.assistant}:'
         else:
             return f'</s>{self.user}: {prompt} {self.assistant}:'
 
     @property
     def stop_words(self):
+        """Return the stop-word's token ids."""
```
Comment (AllentDan) [inline lmdeploy/model.py:34]: stop-word's -> stop-words'

File: lmdeploy/model.py [hunk 6]
```diff
@@ -44,20 +67,33 @@ def get_prompt(self, prompt, sequence_start=True):
...
+            sequence_start (bool): indicator for the first round chat of a
+               session sequence
+        Returns:
+            str: the concatenated prompt
+        """
         return prompt
 
     @property
     def stop_words(self):
+        """Return the stop-word's token ids."""
```
Comment (AllentDan) [inline lmdeploy/model.py:96]: stop-words'

**Review by AllentDan — APPROVED** [review]

**Review by grimoire — APPROVED** [review]

**Pull request merged**